examples/nodejs_etcd_operator/README.md

```diff
@@ -16,31 +16,46 @@
 Test application : https://github.com/akashshinde/node-todo.git
 ![](https://i.imgur.com/WGQZ1nj.png)
 
-4. Create SBR.
+Note: This example assumes the app is deployed using a K8s Deployment if using a OCP DeploymentConfig change the group and resource in the applicationSelector to
+
+```
+    group: apps.openshift.io
+    version: v1
+    resource: deploymentconfigs
+```
+
+4. Add the labels to the deployment or deployment config
+
+```
+oc label deployment node-todo-git 'connects-to=etcd' 'environment=demo'
+```
+
+5. Create SBR.
 ```yaml
 apiVersion: apps.openshift.io/v1alpha1
 kind: ServiceBindingRequest
 metadata:
-  name: binding-request
+  name: node-binding-request
 spec:
   applicationSelector:
-    matchLabels:
-      connects-to: etcd
-      environment: demo
-    group: apps.openshift.io
+    labelSelector:
+      matchLabels:
```

Comment:
Hi @noelo . Thanks for the PR. I believe removing the matchlabels from the ServiceBindingRequest,SBR, completely would be better since we are adding it in step 4.

Reply:
Step 4 is adding the labels to the deployment. I just tried the SBR without the matchLabels and it does work. Do you want that removed ?

Reply:
@Avni-Sharma On step 4, we are adding labels to the deployment.
Here, the labelSelector is mentioned in SBR.
Both are needed.

Reply:
Yeah, either or works but I don't think both are needed- labels and resourceRef. It is more convenient to just have a resourceRef. We can have other examples where we are actually using multiple apps via labels. But having both is indeed redundant